lib/cursor.js

```diff
@@ -171,6 +171,27 @@ Cursor.prototype.cursorSkip = function() {
   return this.cursorState.skip;
 };
 
+Cursor.prototype._endSession = function(options, callback) {
+  if (typeof options === 'function') {
+    callback = options;
+    options = {};
+  }
+  options = options || {};
+
+  const session = this.cursorState.session;
+
+  if (session && (options.force || !session.isExplicit())) {
+    this.cursorState.session = undefined;
+    session.endSession(callback);
+    return true;
```

Comment:
why does this method return a boolean? Actually, this might be a good time to bring up the question: why isn't this method documented at all 😄

Reply:
As with the above comment, the boolean value is used [here](https://github.com/mongodb/node-mongodb-native/pull/1668/files#diff-14b597bc6604e1b7ec69ae786619e979R217). As for documentation, I consider it a "private" method. But I can still add a `/** @private */` or `/** @ignore */`.